osquery/tables/applications/browser_utils.cpp

```diff
@@ -54,32 +54,33 @@ std::vector<ChromeUserExtensions> chromeExtensionPathsByUser(
   std::vector<ChromeUserExtensions> extensionPathsByUser;
 
   for (const auto& row : users) {
-    if (row.count("uid") > 0 && row.count("directory") > 0) {
-      // For each user, enumerate all of their chrome profiles.
-      std::vector<std::string> profiles;
-      for (const auto& chromePath : chromePaths) {
-        fs::path extension_path = row.at("directory") / chromePath;
-        if (!resolveFilePattern(extension_path, profiles, GLOB_FOLDERS).ok()) {
-          continue;
-        }
-
-        // For each profile list each extension in the Extensions directory.
-        for (const auto& profile : profiles) {
-          std::vector<std::string> unversionedExtensions = {};
-          listDirectoriesInDirectory(profile, unversionedExtensions);
+    if (row.count("uid") == 0 || row.count("directory") == 0) {
+      break;
```

Comment:
```suggestion
      continue;
```

Reply:
This was originally a big nested if.
```
 for (const auto& profile : profiles) {
    if (...) {
    ...
   }
}
```

I replaced it with an early return, so that should be `break` right?

Reply:
<snort> no, it is `continue` oops.